eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelTypes.scala

```diff
@@ -200,7 +203,20 @@ final case class LocalParams(nodeId: PublicKey,
                              isFunder: Boolean,
                              defaultFinalScriptPubKey: ByteVector,
                              globalFeatures: ByteVector,
-                             localFeatures: ByteVector)
+                             localFeatures: ByteVector) {
+
+  def fundingPubKey(keyManager: KeyManager) = keyManager.fundingPublicKey(fundingKeyPath)
```

Comment:
This doesn't seem necessary and breaks consistency with other pubkey generators.